sources/model/GGRealTimeMontior.m

```diff
@@ -695,7 +695,19 @@ - (void)sendWatchDriverWithDriverUUID:(nonnull NSString *)uuid
     
     
 }
+- (void)sendCustomerSuccessEventWithCustomerAccessToken:(nonnull NSString *)customerAccessToken
+                                  customerId:(nonnull NSString *)customerId
+                                        completionHandler:(nullable SocketResponseBlock)completionHandler {
+    
+    NSMutableDictionary *params = [NSMutableDictionary dictionaryWithObjectsAndKeys:
+                                   self.developerToken, @"dev_token",
+                                   customerId,@"customer_id",
```

Comment:
cool though see my other comments regarding this that it is probably a valid scenario that it will be nil, os might want to handle this differently